src/schema/tests/test_SchemaMigMgr.py

```diff
@@ -158,7 +159,10 @@ def test_migrate(self):
         class TestMigMgr(SchemaMigMgr):
             def __init__(self, current_version, scripts_dir):
                 self._current_version = current_version
-                super().__init__(scripts_dir)
+                super().__init__(scripts_dir, "connection-uri")
+
+            def _connect(self, connection: str) -> None:
+                self.connection = None
```

Comment:
I think `_connect()` should return a connection instead of setting `self.connection`?